packages/@react-aria/table/src/useTableColumnResize.ts

```diff
@@ -244,24 +251,15 @@ export function useTableColumnResize<T>(props: AriaTableColumnResizeProps<T>, st
     resizerProps: mergeProps(
       keyboardProps,
       {...moveProps, onKeyDown},
-      pressProps
+      pressProps,
+      {style: {touchAction: 'none'}}
     ),
     inputProps: mergeProps(
       visuallyHiddenProps,
       {
         id,
-        onFocus: () => {
-          let resizeOnFocus = !!triggerRef?.current;
-          if (resizeOnFocus) {
-            // useMove calls onMoveStart for every keypress, but we want resize start to only be called when we start resize mode
-            // call instead during focus and blur
-            startResize(item);
-            state.tableState.setKeyboardNavigationDisabled(true);
-          }
-        },
         onBlur: () => {
           endResize(item);
-          state.tableState.setKeyboardNavigationDisabled(false);
         },
```

Comment:
The changes in this file are basically to not assume that if a `triggerRef` is provided that the resizer isn't visible. In the docs example, we have a menu but also the resizer is always visible. Instead, we store whether the resizer itself was focused when resizing starts. If so, focus stays on the resizer when resizing ends, otherwise it goes to the triggerRef.

Reply:
I'll have to go back and revisit the original PR where this assumption was first made, forget if there was a particular reason we made that assumption. This does look like it works for all the various menu with/without always rendered resizer though, so thats good